project/src/main/webapp/index.html

```diff
@@ -2,18 +2,85 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Workout</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="side-nav-menu">
+		<button class="side-toggle" onclick="openNav()"><img src="https://img.icons8.com/ios-filled/35/000000/menu-rounded.png"/></button>
+		<div class="side-menu" id="side-menu">
+		  <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+          <a href="index.html">Home</a>
+          <br>
+          <a href="playlist.html">Playlist</a>
+          <br>
+          <a href="index.html">Favorites</a>
+
+		  <div class="dark-mode-toggle">
+		  	<button class="dark-toggle" onclick="darkMode()"><img src="https://img.icons8.com/material-outlined/40/000000/partly-cloudy-night.png"/></button>
+		  </div>
+		</div>
```

Comment:
Can you fix the whitespace/alignment in this section? It looks like you repeat this code in each file (I can go over how to not need to do that later if we have time, but it's fine for now), so please fix it in those places as well.